src/commands/message/view-message-sent-date.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    ApplicationCommandType,
+    RESTPostAPIContextMenuApplicationCommandsJSONBody,
+} from 'discord-api-types/v10';
+import { MessageContextMenuInteraction, PermissionString } from 'discord.js';
+import { RateLimiter } from 'discord.js-rate-limiter';
+import { DateTime } from 'luxon';
+
+import { EventData } from '../../models/internal-models.js';
+import { Lang } from '../../services/lang.js';
+import { InteractionUtils } from '../../utils/interaction-utils.js';
+import { Command, CommandDeferType } from '../command.js';
+
+export class ViewMessageSentDate implements Command {
+    public metadata: RESTPostAPIContextMenuApplicationCommandsJSONBody = {
+        type: ApplicationCommandType.Message,
```

Comment:
Can we specify this on the chat commands as well?

Reply:
We can but it defaults to ApplicationCommandType.Chat when it isn't given